doc/portfile.7

```diff
@@ -1589,8 +1589,9 @@ Target passed to
 .Sy Example:
 .Dl build.target all-src
 .It Ic use_xcode
-Specifies dependency on Xcode compilers. Restricts Xcode compilers on trace mode
-when set to no.
+Specifies dependency on Xcode compilers. Restricts Xcode compilers on trace
+mode when set to no. This option is overwritten to yes when build.type is
```

Comment:
As mentioned on a different commit, this should probably indicate that only the default changes and developers can still set this value.